src/moonlink_service/src/otel/service.rs

```diff
@@ -70,6 +77,35 @@ pub async fn start_otel_service(
     Ok(())
 }
 
+pub(crate) fn initialize_opentelemetry_meter_provider() -> Result<()> {
+    let otel_exporter = opentelemetry_otlp::MetricExporter::builder()
+        .with_http()
+        .with_endpoint(DEFAULT_HTTP_OTEL_ENDPOINT)
+        .with_protocol(Protocol::HttpBinary) // send protobuf message
+        .build()?;
+
+    let stdout_exporter = opentelemetry_stdout::MetricExporter::builder().build();
```

Comment:
Oh don't use stdout exporter at production, it should be abled by option.

Reply:
For example, for the service start option, it could contain several options on otel: (1) no otel emitted; (2) stdout exporter; (3) to some otel endpoint.